icons/breeze/make-symlinks.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+basedir=$1
+for x in 32 48
+do
+    while read -r line; do
+        read -r -a file <<< "$line"
+        ln -sf ${file[0]} "${basedir}/apps/$x/${file[1]}"
+    done <<< `grep -v '^#' ${x}.txt`
```

Comment:
The `.txt` suffix is usually used for text documents, here I'd use something else, e.g. `.files` or something like that.